faculty-affiliations.csv

```diff
@@ -8239,25 +8239,30 @@ ThanhVu Nguyen , University of Nebraska
 Vinodchandran Variyam , University of Nebraska
 Witawas Srisa-an , University of Nebraska
 Ying Lu , University of Nebraska
-Aslan Munir , University of Nevada
 David Feil-Seifer , University of Nevada
 Dongfang Zhao , University of Nevada
 Dwight D. Egbert , University of Nevada
 Eelke Folmer , University of Nevada
+Engin Arslan , University of Nevada
 Feng Yan 0001 , University of Nevada
 Frederick C. Harris Jr. , University of Nevada
 George Bebis , University of Nevada
 Hung Manh La , University of Nevada
+Hui Lin , University of Nevada
```

Comment:
Please update name to point to actual person on DBLP by adding the appropriate suffix (e.g., 0001).

See http://dblp.uni-trier.de/pers/hd/l/Lin:Hui